src/actions/account.js

```diff
@@ -139,4 +61,29 @@ export const { switchAccount } = createActions({
    SWITCH_ACCOUNT: wallet.selectAccount.bind(wallet)
 })
 
-export const { loginPending, loginSuccess, loginError } = createActions('LOGIN_PENDING', 'LOGIN_SUCCESS', 'LOGIN_ERROR')
+export const { refreshAccount, refreshUrl, resetAccounts } = createActions({
+   REFRESH_ACCOUNT: [
+      wallet.loadAccount.bind(wallet),
+      (loader) => ({ 
+         loader,
```

Comment:
how does this work? Is it generating some `meta` for action? I think we need to standardize the way we pass `meta` to allow generic loader/error handling.